README.md

```diff
@@ -2,4 +2,6 @@
 
 **Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it
+
+I would like to thank Muhammad Sesay for allowing to use his code for the review.
```

Comment:
Great job so far on the project just a few minor changes left. Your readme file is not descriptive. Please add descriptive README file to your project - please use this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Happy coding

Reply:
- This is a polite, well-explained, and easy-to-understand comment. Nice work! :clap: :clap:

Reply:
Also here is a suggestion you can add a live version link of your page to the README.md file for your professional project using [github pages](https://pages.github.com/). Here are some [detailed instructions](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) on how to do so.

Happy coding!